app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/DRecordDetailsFragment.kt

```diff
@@ -74,8 +81,27 @@ class DRecordDetailsFragment : Fragment() {
             putBoolean("isSubFragment", true)
```

Comment:
For the different String Navigation argument key, It would be better to write them somewhere as public Constants, as it is used at least two times in the app.

For example, inside the sender class, the class which launch the navigation.